.all-contributors.md

```diff
@@ -0,0 +1,10 @@
+This list of contributors is generated automatically by [All Contributors](https://allcontributors.org/). The output used in production is the JSON format, this file is not used.
```

Comment:
Can this file be removed if it is not used?

Reply:
Unfortunately not.